test/eradius_config_SUITE.erl

```diff
@@ -43,7 +43,7 @@ config_1(_Config) ->
                       { {"NAS1", [arg1, arg2]},
                           [{"10.18.14.2/30", <<"secret1">>}]},
                       { {"NAS2", [arg1, arg2]},
-                          [{"10.18.14.3", <<"secret2">>, [{nas_id, <<"name">>}]}]}
```

Comment:
@thz it tests in `config_2/1` at [eradius_config_SUITE.erl:86](https://github.com/travelping/eradius/blob/master/test/eradius_config_SUITE.erl#L86), [eradius_config_SUITE.erl:87](https://github.com/travelping/eradius/blob/master/test/eradius_config_SUITE.erl#L87) and [eradius_config_SUITE.erl:89](https://github.com/travelping/eradius/blob/master/test/eradius_config_SUITE.erl#L89)